apps/web/app/routes/posts/route.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import type { Route } from "./+types/route";
+import { dehydratedQueryResponse } from '@/web/query/util';
+import { meQueryOptions } from '@/web/query/user.query';
+import PostCard from './components/post-card';
+import PostCardList from './components/posts-list';
+
+type Props = {}
+export async function loader({ request }: Route.LoaderArgs) {
+  return dehydratedQueryResponse(request, meQueryOptions);
```

Comment:
Why are you using `meQueryOptions` here? Aren't you supposed to dehydrate the posts query instead?